### DISCUSSION TITLE
Button in Search Results webpart to trigger a Power Automate flow

### DISCUSSION THREAD

**Discussion by ileana2019** [microsoft-search/pnp-modern-search#1576] [Q&A]
Hello,
Is it possible with the Search Results webpart to have a button that triggers a Power Automate flow? Like with the SharePoint list views. 
Thank you!
![image](https://user-images.githubusercontent.com/55545989/144206817-5419cea9-eff4-4c2e-960f-8fc58d42d6a8.png)

**Comment by FranckyC** [comment]
Hi @ileana2019, yes you can easily do that with the extensibility model and web components. See https://microsoft-search.github.io/pnp-modern-search/extensibility/custom_web_component/

**Reply by ileana2019** [reply]
Hi @FranckyC,
Together with my team, we tried to implement the custom web component just by deploying the [pnp - Extensibility Library Demo Project](https://github.com/microsoft-search/pnp-modern-search-extensibility-samples/tree/main/search-extensibility-samples). Do we really need a custom web component, or we can just add a column to the results layout, having the handlebar expression: <a href=.....</a>. Does the Power Automate instant flow needs the trigger: “When a HTTP request is received” in order to connect to the PnP webpart  row item?  Or how do we connect the button and the Power Automate flow?
Thank you!
Regards,
Ileana

**Reply by wobba** [reply]
If you use `<a href="flow url">` that will work.

**Reply by ileana2019** [reply]
Hi @wobba, Thanks for the reply! And do you know if the Power Automate instant flow needs the trigger: “When a HTTP request is received” in order to connect to the PnP results webpart row item? 
Thank you!

**Reply by wobba** [reply]
Any Flow related question is outside of my expertise these days :)
